packages/generators/utils/plugins.ts

```diff
@@ -1,11 +0,0 @@
-/**
```

Comment:
Are we using this file somewhere?

Reply:
No, we used it inside the `init-generator` ([link](https://github.com/webpack/webpack-cli/blob/master/packages/generators/init-generator.ts#L12)) calling it `getDefaultPlugins`

I've just moved this code into the default production configuration ([here](https://github.com/misterdev/webpack-cli/blob/misc/improve-init-generator/packages/generators/init-generator.ts#L101))

Reply:
Okay. Could you abstract that part into a func? Looks messy

Reply:
I've moved it to a new file [`generators/utils/webpackConfig`](https://github.com/misterdev/webpack-cli/blob/misc/improve-init-generator/packages/generators/utils/webpackConfig.ts) . I chose this name so that it can be possibly used for other configuration utilities (eg. defaultConfiguration)